src/inmanta/protocol/auth/default_policy.rego

```diff
@@ -0,0 +1,159 @@
+package policy
+
+# Write the information about the endpoint into a variable
+# to make the policy easier to read.
+endpoint_data := data.endpoints[input.request.endpoint_id]
+
+# The environment used in the request
+request_environment := input.request.parameters[endpoint_data.environment_param] if {
+    endpoint_data.environment_param != null
+} else := null
+
+# Don't allow anything that is not explicitly allowed.
+default allowed := false
```

Comment:
I only notice this now that I'm referencing the language documentation. Any reason we use `allowed` instead of `allow` used in the language examples?

Reply:
Not really. Do you prefer allow?

Reply:
I have a tiny preference for it, solely because of the chance that someone who isn't familiar with open-policy agent would go to the rego docs and start copy-pasting something together.